packages/pluginutils/package.json

```diff
@@ -75,12 +75,17 @@
       "!**/types.ts"
     ]
   },
-  "module": "dist/index.es.js",
+  "exports": {
+    "require": "./dist/cjs/index.js",
+    "import": "./dist/es/index.js"
+  },
+  "module": "./dist/es/index.js",
   "nyc": {
     "extension": [
       ".js",
       ".ts"
     ]
   },
+  "type": "commonjs",
```

Comment:
This will treat `dist/es/index.js` as CommonJS - are you sure that is the intention here?

If not it is probably advisable to change `dist/es/index` to use the `.mjs` extension, or switch things around to the `.cjs` extension.

Nested package.json files can form sub-boundaries within packages, but its not necessarily recommended.

Reply:
Nested package.json is what I do here
https://github.com/rollup/plugins/pull/419/files#diff-bdf5352c3e711371ce12e70c4f962f4dR27-R38

Reply:
Ah right, didn't see that!